py3status/modules/khal_calendar.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+Displays upcoming khal events.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 60)
+    config_path: Path to khal configuration file. The default None resolves to /home/$USER/.config/khal/config (default 'None')
```

Comment:
`specify a config path, otherwise default config path (default None)` should fix it.